pkgs/applications/window-managers/sway/wrapper.nix

```
@@ -23,6 +24,7 @@ let
        ${extraSessionCommands}
        export _SWAY_WRAPPER_ALREADY_EXECUTED=1
      fi
+     '' + lib.optionalString dbusSupport ''
```

```suggestion
   '' + lib.optionalString dbusSupport ''
```

Nit: IMO it's more readable that way (and AFAIK that's the usual style).

---

Done, thanks (will push momentarily).  Yes, I agree, it's more readable that way.